src/ui.container-modal/UModal.vue

```diff
@@ -321,7 +321,7 @@ function preventOverlayFromScrolling(newValue) {
   if (newValue) {
     document.body.style.overflow = "hidden";
   } else {
-    document.getElementById(`${props.id}`).style.overflow = "hidden";
```

Comment:
please fix same issue in all places at once:
![image](https://github.com/user-attachments/assets/c9d14d22-509f-4252-a364-f463f5f5a46f)